jax/experimental/jax2tf/tests/primitive_harness.py

```diff
@@ -163,21 +171,24 @@ def __init__(self,
                jax_unimplemented: Sequence["Limitation"] = (),
                **params):
     """See class docstring."""
-    self.group_name = group_name
-    self.name = name
+    self.group_name = self.sanitize_name(group_name)
+    self.name = self.sanitize_name(name)
+    self.fullname = self.name if self.group_name is None else f"{self.group_name}_{self.name}"
     self.fun = fun  # type: ignore[assignment]
     self.arg_descriptors = arg_descriptors
     self.rng_factory = rng_factory  # type: ignore[assignment]
     self.jax_unimplemented = jax_unimplemented
     self.dtype = dtype
     self.params = params
 
+  def sanitize_name(self, s: str) -> str:
+    res = s.translate(kSanitizeNameTranslateTable)
+    # assert re.match(r"^[\w|:|\+|\-|\.|\[|\]|\\|/]+$", res), res
```

Comment:
missed?

Reply:
Sorry, what is missed? The commented-out assert? I have removed it now.